frontend/src/app/AppLauncher.tsx

```diff
@@ -112,13 +112,13 @@ const AppLauncher: React.FC = () => {
       const action: ApplicationAction = {
         label: link.spec.text,
         href: link.spec.href,
-        image: <img src={link.spec.applicationMenu.imageURL} alt="" />,
+        image: <img src={link.spec.applicationMenu?.imageURL} alt="" />,
```

Comment:
You should mark this type attribute as optional in the `type` right?

Reply:
This was the minimal fix I found. If I make `applicationMenu` optional in the type, then we may have to make `label` in `Section` in `AppLauncher` optional too. Is it okay to do that?

Reply:
Don't half fix this solution, if you see that the code is not right, try to refactor and give the proper fix.

The issue itself is that we have an incorrect `ConsoleLinkKind` type, you can check the spec in your cluster following this path `api-resource/cluster/console.openshift.io~v1~ConsoleLink/schema`. As you can see, there are several attributes missing for the type, you should update it both in the **backend** and the **frontend**.

And then if that type make some issues in our code, we need to refactor the code according to the type.